widgetastic_manageiq.py

```diff
@@ -878,3 +878,37 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class ToolBarViewSelector(View):
+    """ represents toolbar's view selector control ("Grid View", "Tile View", etc)
+
+        .. code-block:: python
+        @View.nested
+        class view_selector(ToolBarViewSelector):  # NOQA
+
+        some_view.view_selector.select('Tile View')
+        some_view.view_selector.selected
+    """
+    ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
+    BUTTONS = './/button'
+
+    @property
+    def _view_buttons(self):
+        br = self.browser
+        return [Button(self, title=br.get_attribute('title', btn)) for btn
+                in br.elements(self.BUTTONS)]
```

Comment:
Not a big fan of doing the `Button(self, ...)` because you are not supposed to but this `list`-like functionality needs to be added to widgetastic, right?